tools/debug/debugsymbolviewer.py

```diff
@@ -228,6 +228,13 @@ def print_stack(log_file):
 						continue
 				continue
 
+			if 'Code asserted in IRQ state!' in line:
+				print('[Interrupt Stack]')
+			if 'Code asserted in nested IRQ state!' in line:
+				print('[Interrupt Stack]')
+			if 'User stack:' in line:
+				print('[User stack]')
```

Comment:
Should we not check for `Code asserted in normal thread!` here?

Reply:
I have modified the code to add this log in case of assertion in any stack